htdocs/core/menus/standard/auguria.lib.php

```diff
@@ -240,7 +240,9 @@ function print_left_auguria_menu($db,$menu_array_before,$menu_array_after,&$tabM
 		print '<div class="menu_titre" id="menu_titre_logo"></div>';
 		print '<div class="menu_top" id="menu_top_logo"></div>';
 		print '<div class="menu_contenu" id="menu_contenu_logo">';
-		print '<div class="center"><img title="'.dol_escape_htmltag($title).'" alt="" src="'.$urllogo.'" style="max-width: 80%"></div>'."\n";
+		print '<a class="center" href="'.DOL_URL_ROOT.'/index.php?mainmenu=home&amp;leftmenu=" alt="'.dol_escape_htmltag($langs->trans("Home")).'" title="'.dol_escape_htmltag($langs->trans("Home")).'">';
+		print '<div class="center"><img src="'.$urllogo.'" style="max-width: 80%"></div>'."\n";
```

Comment:
The alt and title should actually be on this line on the img tag. The a tag does not have alt and title attributes.